setup.py

```diff
@@ -71,10 +71,12 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Topic :: Text Processing :: Markup :: HTML'
     ],
-    packages=['jinja2', 'jinja2.testsuite', 'jinja2.testsuite.res'],
-    install_requires=['MarkupSafe'],
+    packages=['jinja2'],
+    install_requires=[
```

Comment:
To be clear, test dependencies can be listed in the toxfile (and tox will take care of installing these dependencies).